AppController/lib/nginx.rb

```diff
@@ -356,69 +351,68 @@ def self.create_datastore_server_config(all_private_ips, proxy_port)
     
 server {
     listen #{DatastoreServer::LISTEN_PORT_NO_SSL};
-    root /root/appscale/AppDB/;
+    root   /root/appscale/AppDB/;
```

Comment:
Is there a constant somewhere where we defined where appscale has been installed? I think during build is APPSCALE_HOME, can we use that instead of /root/appscale?